db/migrate/20230303_create_students.rb

```diff
@@ -0,0 +1,13 @@
+class CreateStudents < ActiveRecord::Migration
+    def change
+        create_table :students do |t|
+            t.string :title
+            t.string :first_name
+            t.string :middle_name
+            t.string :last_name
+            t.string :email
+            t.date :birth_date
+            t.string :gender
+        end
+    end
+end
```

Comment:
it's best practice to use 2 spaces for indentation.
```suggestion
class CreateStudents < ActiveRecord::Migration
  def change
    create_table :students do |t|
      t.string :title
      t.string :first_name
      t.string :middle_name
      t.string :last_name
      t.string :email
      t.date :birth_date
      t.string :gender
    end
  end
end
```